src/Umbraco.Core/Services/PackagingService.cs

```diff
@@ -762,7 +762,7 @@ private void UpdateContentTypesProperties(IContentType contentType, XElement gen
         private IContentType UpdateContentTypesStructure(IContentType contentType, XElement structureElement)
         {
             var allowedChildren = contentType.AllowedContentTypes.ToList();
-            int sortOrder = allowedChildren.Any() ? allowedChildren.Last().SortOrder : 0;
+            int sortOrder = allowedChildren.Any() ? allowedChildren.LastOrDefault().SortOrder : 0;

```

Comment:
This change is pretty much redundant as the Last() child is always going to be find if there are Any() children.